e2e/js_image_docker/WORKSPACE

```diff
@@ -31,6 +31,13 @@ load("@npm//:repositories.bzl", "npm_repositories")
 
 npm_repositories()
 
+# TODO: remove when rules_?  upgrades to modern version supporting
```

Comment:
An indirect dep of maybe rules_docker includes an older version of gazelle which breaks with the new version of rules_go.

Reply:
Work-around looks good to me